app/app.js

```diff
@@ -0,0 +1,13 @@
+import express from "express";
+import config from "./config/config.js";
+import meteorRouter from "./delivery/meteor_routes.js";
+
+const app = express();
+const port = config.port;
+
+app.use(express.json());
+app.use(meteorRouter);
+
```

Comment:
add `errorHandler` please somewhere there

here you can find a little bit more information https://medium.com/@mohsinogen/simplified-guide-to-setting-up-a-global-error-handler-in-express-js-daf8dd640b69

Reply:
Done